NAMESPACE

```diff
@@ -30,14 +30,24 @@ export(get_measurements)
 export(get_paleogeography)
 export(get_sections)
 export(get_stats)
+export(get_tiles)
 export(get_units)
 importFrom(curl,nslookup)
+importFrom(data.table,rbindlist)
 importFrom(geojsonsf,geojson_sf)
 importFrom(httr,GET)
 importFrom(httr,content)
 importFrom(httr,http_error)
 importFrom(jsonlite,fromJSON)
+importFrom(protolite,read_mvt_sf)
+importFrom(sf,"st_geometry<-")
```

Comment:
Is this a mistake? I've never seen "<-" included in importFrom before?

Reply:
Not a mistake, this is a setter method that can be used to set the name of an sf object, like so:
```r
st_geometry(sf) <- geometry_name
```